spec/acceptance/nodesets/oel.yml

```diff
@@ -0,0 +1,41 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
+-%>
+HOSTS:
+  server-el7:
+    roles:
+      - server-el7
+      - default
+    platform: el-7-x86_64
+    box: onyxpoint/oel-7-x86_64
+    hypervisor: <%= hypervisor %>
+    yum_repos:
+      epel:
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch'
+        gpgkeys:
+          - https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-$releasever
+
+  server-el6:
+    roles:
+      - server-el6
+    platform: el-6-x86_64
+    box: onyxpoint/oel-6-x86_64
+    hypervisor: <%= hypervisor %>
+    yum_repos:
+      epel:
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-6&arch=$basearch'
+        gpgkeys:
+          - https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-$releasever
+
```

Comment:
No OEL 8 box yet?

Reply:
@lnemsick-simp There's a `generic` one but not an `onyxpoint` one so I'm not going down that road yet.

Reply:
Well, I'll give it a shot and see what happens anyway.

Reply:
Appeared to work! PR updated.